src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
Sorry, this wasn't an intentional formatting change but just a consequence of me editing in VSCode with `trimTrailingWhitespace = true`.  I can fix it.  Will submit an update later tonight.